src/main.js

```diff
@@ -129,13 +133,26 @@ $(document).ready(() => {
             if (isSidebarPinned()) toggleSidebar();
             else togglePin();
           } else if (isSidebarVisible()) {
+            // @TODO(add current working branch to local store for a certin period of time)
+            // For now this could be not a good approch because when
+            // The page is force reloaded the current branch being navigated will be lost, but for now it works.
+            if (isChangedBranch && typeof branch !== 'undefined') {
+                repo.branch = branch;
```

Comment:
is it possible to save branch in memory so that it wouldn't be lost if somewhere else executes `tryLoadRepo`